jekyll/_components/tables.md

```diff
@@ -2,17 +2,142 @@
+
+<div markdown="1">
+Tables are used when data needs to be presented in a series of rows which may also posses any amount of columns. For large quantities of data it is recommended to include a button column header allowing users to sort the data by that specific column, as seen in the Network panel
+</div>
+</div>
+
+
 ## Types
+
+### Console tables
+
+<div class="grid-2" markdown="1">
+![Console table example](../images/components/tables/console-table-small.png)
+
+<div markdown="1">
+The console can display a tablular representation of an Array or Object. e.g. `console.table(["apples", "oranges", "bananas"]);`
+
+Console table columns are not sortable or resizable.
+</div>
+</div>
+
+### Network and Application panel
     
+<div class="grid-2" markdown="1">
+![Network table timeline example](../images/components/tables/table-timeline.png)
+
+<div markdown="1">
+The Network panel table features column sorting and row highlighting on hover. the table can contain a rich variety of data including text, images (icons: `16px` x `16px`) and timeline interface elements like graphs and vertical rules (pictured).
+
+Filter controls are provided in the form of buttons for specific resource types and a filter input field for filtering URLs.
+</div>
+</div>
+
 ## Styles
 
+Table rows alternate their background color, the subtle "zebra" effect assists in legibility:
+
+### Dark mode
+<div class="grid-2" markdown="1">
+![Usage Example of a Checkbox as used in Firefox Options](../images/components/tables/table-grid-dark.png)
+
+<div markdown="1">
+Odd rows: Grey 80 `#2a2a2e`
+
+Even rows: `rgba(255,255,255,0.05)`
+
+Column header: Grey 85 `#1b1b1d`
+
+Borders/splitters: `#3c3c3d`
+
+Text: Grey 40 `#b1b1b3`
+
+Selected row: `#204E8A`
+</div>
+</div>
+
+### Light mode
+<div class="grid-2" markdown="1">
+![Usage Example of a Checkbox as used in Firefox Options](../images/components/tables/table-grid-light.png)
+
+<div markdown="1">
+Odd rows: white `#ffffff`
+
+Even rows: `rgba(0,0,0,0.05)`
+
+Column header: Grey 10 `#f9f9fa`
+
+Borders/splitters: Grey 25 `#e0e0e2`
+
+Text: Grey 70 `#38383d`
+
+Selected row: Blue 55 `#0074e8`
+</div>
+</div>
+
+
+<div class="grid-2" markdown="1">
+![Table size and font properties](../images/components/tables/table-size-properties.svg)
+<div markdown="1">
+Font size: `11px`
+
+Row height: `24px`
+
+Line height: `24px`
+</div>
+</div>
+
 ## Behaviours
+
+### Sorting
+<div class="grid-2" markdown="1">
+![Table Column headers](../images/components/tables/table-header-button.png)
+
+<div markdown="1">
+Tables can posses sortable column headers, clicking these buttons will toggle the sorting of the column they relate to. In the image, clicking the "Size" button header toggles the Size column from largest to smallest and from smallest to largest.
+
+Selected column header (Light mode): Blue 55 `#0074e8`
+
+Selected column header (Dark mode): `#204E8A`
```

Comment:
Do we not have a Photon colour for the dark mode selected column header? Can we find one that's close and say that's the one we should use? (And use the same photon colour for the selected row above, of course. 😉)

Reply:
Having compared, the best matches are Blue 60 or 70. By eye, Blue 70 is a better match for the current #204E8A so I'm proceeded with that.